### PR TITLE
HA master: configurable replica name during kube-down.

### PR DESCRIPTION
**Pull request by jszczepkowski**
HA master: configurable replica name during kube-down.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37381)

### CODE REVIEW

**Comment by jszczepkowski** [comment]
Part of https://github.com/kubernetes/features/issues/48

File: cluster/gce/util.sh [hunk 1]
```diff
@@ -1328,7 +1328,10 @@ function kube-down() {
     done
   fi
 
-  local -r REPLICA_NAME="$(get-replica-name)"
+  local REPLICA_NAME="${KUBE_REPLICA_NAME:-}"
```
Comment (gmarek) [inline cluster/gce/util.sh]: local REPLICA_NAME="${KUBE_REPLICA_NAME:-$(get-replica-name)}"

Comment (jszczepkowski) [inline cluster/gce/util.sh]: done

**Comment by jszczepkowski** [comment]
@k8s-bot kops aws e2e test this

**Pull request merged**